open-ssid-list.txt

```diff
@@ -1,66 +1,156 @@
+#TELUSDirect
 #WiFi@Changi
-AIRPORT
 .Venice Airport Free WiFi
+A&W Free Wi-Fi
+ADPGUESTWIFI
+AIRPORT
+ASUS
 AirPort Express
 Airport_FreeInternet
 Apple Demo
+Apt3Guest
 Auckland Airport
 BAYMONT
+BELL762-10-guest
+BELLWIFI@MCDONALDS
 BELL_WIFI
+BK Guest
+BMOguest
+BSA Guest
+BayRentalsGuest
+Blazemarketing
 Boingo Hotspot
+Boneheads
+Boston Pizza
+Burger King (Public)
 Caesars_Resorts
+Canadian Tire
 Caravel Conferences
 Caravel Wireless
 Chick-fil-A WiFi
 Chili's
+CoastTire
 CoffeeBeanWifi
 CrossCountryWiFi
+DAVIDsTEA - Free Internet
 DOCOMO@Hilton Meetings
 DXB Free WiFi
+Dal-WPA2
+Dental-Guest
 DomoticaSys
+ES-Guest
+GH_Cottages
+GOEASY_GUEST
 GUEST
 Gatwick FREE Wi-Fi
+George's Bubble Tea
 Google Starbucks
+GrecoFreeWiFi
 Guest
+Guest Access
+Guest-Wireless
 Guest@Balalaika
 Guest@Hilton
 HHonors
+Halifax Public Libraries
+Hanatech-Guest
+Hawthorn WIFI
+Hewitt_Guest
 Hilton Honors
 HolidayInn
 HolidayInn-GuestRoom
 HolidayInn_Guest
+HotSpot@NSCC
+IMV-Guest
+IWK-Guest
+Ironstone Guest
+Irving-guest
+JACK ASTOR'S GUEST
+JA_PUBLIC
+KFC Guest
 MARRIOTT_GUEST
+MAYGARDENWIFI
 MSFTOPEN
+MTSWireless
 Macdonald Manchester WiFi
+Massage Addict Guest
 MaundGuest
+MayGardenGuest
+Middle Spoon Guest
+Millcove_Guest
+Montana's-Guest
+NOCI-GUEST
+NSGOV-GUEST
+NYCommune
 O2 Wifi
-Qantas-Lounge
```

Comment:
Why did you remove `Qantas-Lounge`?

Reply:
Ah, I see it further below

Reply:
Sorry about that, I think that shuffled around when I was trying to merge in my additions.